src/EnergyPlus/WaterCoils.cc

```diff
@@ -2461,6 +2473,10 @@ void SizeWaterCoil(EnergyPlusData &state, int const CoilNum)
             sizerCWDesWaterInTemp.initializeWithinEP(state, CompType, CompName, bPRINT, RoutineName);
             state.dataWaterCoils->WaterCoil(CoilNum).DesInletWaterTemp = sizerCWDesWaterInTemp.size(state, TempSize, ErrorsFound);
 
+            if (state.dataWaterCoils->WaterCoil(CoilNum).UseDesignInletWaterTemp) {
+                state.dataWaterCoils->WaterCoil(CoilNum).DesInletWaterTemp = state.dataWaterCoils->WaterCoil(CoilNum).DesignInletWaterTemp;
+            }
+
```

Comment:
If user specified *Design Inlet Water Temperature*, then this value will be used during sizing. This field is used in Detailed Water Cooling Coil object only.